pkg/collector/corechecks/servicediscovery/module/config_test.go

```diff
@@ -0,0 +1,47 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//go:build test && linux_bpf
+
+package module
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/DataDog/datadog-agent/pkg/config/mock"
+)
+
+func TestConfigIgnoredComms(t *testing.T) {
+	t.Run("via YAML", func(t *testing.T) {
+		mockSystemProbe := mock.NewSystemProbe(t)
+		mockSystemProbe.SetWithoutSource("discovery.ignored_command_names", "dummy1 dummy2")
+
+		discovery := newDiscovery()
+		require.NotEmpty(t, discovery)
+
+		assert.Equal(t, len(discovery.config.ignoreComms), 2)
+	})
```

Comment:
here you don't use yaml at all

Reply:
I've changed this unit test, moved here tests relevant for system-probe configuration from comm_test.go and removed duplicated cases.